Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.ts

```diff
@@ -254,7 +254,8 @@ class BocAutoCompleteReferenceValue //TODO RM-7715 - Make the TypeScript classes
           }
           else
           {
-            this.RestoreIcon();
+            const isEmpty = actualItem.DisplayName === '';
```

Comment:
This breaks semantics. A bit further up in the code, we use `aitem.UniqueIdentifier == this._nullValueString` to test for "empty" semantics.